### DISCUSSION TITLE
Error generating types: "failed to determine PostgREST version for project"

### DISCUSSION THREAD

**Discussion by jimmiejackson414** [supabase/supabase#19758] [Questions]
I'm working to get a Nuxt3/ Supabase project off the ground, but am running in to issues [generating the types](https://supabase.com/docs/reference/javascript/typescript-support).

I have this script in my package.json that I brought over from a previous project:
```
"typegen": "npx supabase gen types typescript --project-id '<project-id>' --schema public > ./types/index.ts && npx better-supabase-types -i ./types/index.ts -f -s",
```

However after a few seconds, I receive the error:
```
failed to retrieve generated types: {"message":"failed to determine PostgREST version for project<project-id>","errorEventId":"d66db2d43c184bae93cce5eb4efca1c4"}
Try rerunning the command with --debug to troubleshoot the error.
error Command failed with exit code 1. 
```
I've also tried the one directly from the docs for a simpler version:
```
supabase gen types typescript --project-id <project-id> > ./types/index.ts
```
Which results in the same error.

I am using the [@nuxtjs/supabase module](https://supabase.nuxtjs.org/get-started), so I'm unsure of it's an issue with the module itself, or if there is a way to set the PostgREST version for this project as the error mentions?

Note: It might also be helpful to know that I'm running Supabase locally

**Comment by jimmiejackson414** [comment]
Well this question didn't last long (classic).

I discovered that it's because my hosted version of Supabase was paused. I assumed that because I am running Supabase locally, it would generate the types from my local instance rather than the hosted instance.

Which begs another question, is there a way to generate them from the locally hosted instance instead?

**Reply by supermar1010** [reply]
```supabase gen types typescript --local```
